package.json

```diff
@@ -23,51 +23,48 @@
     "url": "https://github.com/dojo/cli-create-app.git"
   },
   "scripts": {
-    "prepublish": "grunt peerDepInstall",
+    "build:static:dev": "cpx \"{src,tests}/**/*.{d.ts,ejs}\" dist/dev",
+    "build:static:dev:templates": "cpx \"src/templates/**/*\" dist/dev/src/templates",
+    "build:static:release": "cpx \"src/templates/**/*\" dist/release/templates",
+    "build:cjs": "tsc",
+    "build": "concurrently \"npm:build:*\" && dojo-package",
+    "clean": "rimraf dist coverage",
+    "lint": "tslint -p .",
     "precommit": "lint-staged",
     "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
-    "test": "grunt test"
+    "test": "npm run build && intern",
```

Comment:
We should add `prettier -l {src,tests}/**/*.ts` to test in order to fail the build if files don't comply. It used to be possible to not run precommit if the user already had a git precommit script. Don't know if this has been fixed in husky.